tests/updates.rs

```diff
@@ -46,6 +47,89 @@ fn test_updates_and_merging() {
 
     // Sending
     let mut doc_send = Automerge::new();
+    let mut tx = doc_send.transaction();
+    let (player_id, camera_id, numbers_id) =
+        send(&mut tx, &player_send, &camera_send, &numbers_send);
+    let (commit, _patch_log) = tx.commit();
+    let base_commit = commit.unwrap();
+    // Send the whole document
+    let data = doc_send.save();
+
+    // Receiving
+    let mut doc_receive = Automerge::load(&data).unwrap();
+    let (mut player_receive, mut camera_receive, mut numbers_receive) = receive(&doc_receive);
+    assert_eq!(player_send, player_receive);
+    assert_eq!(camera_send, camera_receive);
+    assert_eq!(numbers_send, numbers_receive);
+
+    // Updating
+    let mut tx = doc_send.transaction();
+    update1(
+        &mut tx,
+        &mut player_send,
+        &mut camera_send,
+        &mut numbers_send,
+        &player_id,
+        &camera_id,
+        &numbers_id,
+    );
+    let (commit, _patch_log) =
+        tx.commit_with(CommitOptions::default().with_message("Updating position"));
+    let _commit_after_update1 = commit.unwrap();
```

Comment:
Can this be removed?

Reply:
Purely talking about the unwrap line. I understand wanting to clarify there's something returned by the `commit_with` function. Should actually be marked as nit anyway, it's nothing blocking